repo/packages/M/msoms/0/marathon.json.mustache

```diff
@@ -27,14 +32,11 @@
       "privileged": true,
       "parameters": [
         {
```

Comment:
never remove anything from a version. Please create a new version of msoms package and add your changes